ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -502,11 +502,22 @@ objects:
         cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
 
         workers=3
-        if [[ "${CLUSTER_VARIANT}" == "compact" ]]; then
+        if [[ "${CLUSTER_VARIANT}" =~ "compact" ]]; then
           workers=0
         fi
-        if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
-          cat > /tmp/artifacts/installer/install-config.yaml << EOF
+        if [[ "${CLUSTER_TYPE}" = "aws" ]]; then
+            subnets="[]"
+            if [[ "${CLUSTER_VARIANT}" =~ "shared-vpc" ]]; then
+              case $((RANDOM % 4)) in
+              0) subnets="['subnet-0d07ce3a7426acc1d','subnet-04ae7cb2c7cfd71c4','subnet-0b48563892410ebe4','subnet-048f8b03469c8101a','subnet-0746d365a980fb743','subnet-0089243a6fc4f351b']";;
+              1) subnets="['subnet-0fa3c14171a676f9e','subnet-00261f3ebc2dd4919','subnet-0f538375294f59f59','subnet-05123206c6c90edbe','subnet-0e0a3d2ae6ba46ccf','subnet-02617c280fb382d63']";;
+              2) subnets="['subnet-0c88e7123edc58283','subnet-0acf2a196c213049f','subnet-099fa9f0051ec7fb1','subnet-0a5fe27ba3df2fce7','subnet-04d6c084feeaafd61','subnet-0f186f37f88825bde']";;
+              3) subnets="['subnet-0e473918bc57753b0','subnet-0c408f513053ccf60','subnet-0694b0c0e2e2920e0','subnet-0b4bba6548eacefab','subnet-09e6fa06f1b867f82','subnet-0f0fbe84bbe8aaa74']";;
```

Comment:
Do we have committed code somewhere to set these up?

Reply:
nope, these just exit in the CI account using CF templates.

Reply:
Probably want a doc somewhere (maybe just the commit message) for "someone blew away our long-lived subnets, how do we recover?".

Reply:
done, added to the commit message 85ff0d8